iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ValidationUtil.java

```diff
@@ -375,32 +378,20 @@ public static void checkRatedU2(Validable validable, double ratedU2) {
         checkRatedU(validable, ratedU2, "2");
     }
 
-    public static ValidationLevel checkSvcRegulator(Validable validable, double voltageSetpoint, double reactivePowerSetpoint,
+    public static ValidationLevel checkSvcRegulator(Validable validable, boolean regulating, double voltageSetpoint, double reactivePowerSetpoint,
                                                     StaticVarCompensator.RegulationMode regulationMode, ValidationLevel validationLevel, ReportNode reportNode) {
-        return checkSvcRegulator(validable, voltageSetpoint, reactivePowerSetpoint, regulationMode, checkValidationLevel(validationLevel), reportNode);
+        return checkSvcRegulator(validable, regulating, voltageSetpoint, reactivePowerSetpoint, regulationMode, checkValidationLevel(validationLevel), reportNode);
     }
 
-    private static ValidationLevel checkSvcRegulator(Validable validable, double voltageSetpoint, double reactivePowerSetpoint,
+    private static ValidationLevel checkSvcRegulator(Validable validable, boolean regulating, double voltageSetpoint, double reactivePowerSetpoint,
```

Comment:
In the `checkPhaseTapChangerRegulation` function, there is a check on (`regulationMode == null`). I think we need the same kind of check here too if we give `regulationMode` a null value in `RegulatingControlMappingForStaticVarCompensator`)